rules/prefer-includes.js

```diff
@@ -26,29 +32,40 @@ const create = context => ({
 	BinaryExpression: node => {
 		const {left, right} = node;
 
-		if (isMethodNamed(left, 'indexOf')) {
-			const target = left.callee.object;
-			const pattern = left.arguments[0];
+		if (!isMethodNamed(left, 'indexOf')) {
+			return;
+		}
+
+		const target = left.callee.object;
 
-			if (right.type === 'UnaryExpression') {
-				const {argument} = right;
+		// Ignore {_,lodash,underscore}.indexOf
+		if (target.type === 'Identifier' && ['_', 'lodash', 'underscore'].includes(target.name)) {
```

Comment:
This shouldn't be hardcoded here. Either (a) convert it to a blacklist configuration property with these values as a default or (b) move the values to a constant at the top of the file.

Reply:
will fix it tomorrow with (b) 

I think it should be whitelist not blacklist, right?

Reply:
The phrase "ignore" is less ambiguous. Maybe something like `ignoreWords`.

Reply:
Good idea, going to get up and fix it now